book/src/themes.md

```diff
@@ -224,6 +224,7 @@ These scopes are used for theming the editor interface.
 | `ui.text`                |                                     |
 | `ui.text.focus`          |                                     |
 | `ui.text.info`           |                                     |
+| `ui.text.whitespace`     | Visible white-space characters      |
```

Comment:
@sudormrfbin I think visible whitespace and indent guides could share a scope namespace. What would you think about `ui.background.{indent-guide,whitespace}` or `ui.virtual.{indent-guide,whitespace}` so they can be configured together and customized separately?

Reply:
Seems like a good idea; I like the `ui.virtual` scope better.